tests/fail/should-pass/cpp20_rwc_syncs.rs

```diff
@@ -1,6 +1,6 @@
 // ignore-windows: Concurrency on Windows is not supported yet.
 // compile-flags: -Zmiri-ignore-leaks
-// error-pattern:
```

Comment:
@oli-obk btw, this was matching the warning that used to be printed when spawning a thread. There was no actual 'error' in this execution. I don't think ui_test should have accepted this.